scripts/ghidra/PatchestryDecompileFunctions.java

```diff
@@ -846,7 +851,20 @@ private Address convertAddressToRamSpace(Address address) throws Exception {
 			if (address == null) {
 				return null;
 			}
-			return ram_space.getAddress(address.toString(false));
+
+			try {
+				// If already in RAM space, just return it
+				if (address.getAddressSpace().getName().equals("ram")) {
+					return address;
+				}
+
+				// Get the numeric offset and create a new address in RAM space
+				long offset = address.getOffset();
+				return program.getAddressFactory().getDefaultAddressSpace().getAddress(offset);
+			} catch (Exception e) {
```

Comment:
I would prefer it if we bubbled exceptions up and handled them all at a more top-level place (possibly halting execution?), rather than swallowing them down in the depths of the pcode extraction logic here.

Reply:
We can consume `AddressOutOfBoundsException` early and throw runtime exception for others. This will not blind us from other possible issues.